src/lib/operators/aggregate_hash.hpp

```diff
@@ -19,14 +19,15 @@
 #include "bytell_hash_map.hpp"
 #include "expression/aggregate_expression.hpp"
 #include "resolve_type.hpp"
-#include "storage/abstract_segment_visitor.hpp"
 #include "storage/reference_segment.hpp"
 #include "storage/value_segment.hpp"
 #include "types.hpp"
 #include "utils/assert.hpp"
 
 namespace opossum {
 
+class SegmentVisitorContext {};
```

Comment:
Is this a forward declaration? Those usually come without curly braces.

Reply:
Or is this because the aggregate is the only one who still needs the visitor? In that case, maybe add a short comment